modules/islandora_oaipmh/README.md

```diff
@@ -0,0 +1,28 @@
+# Islandora OAI-PMH Endpoint
+
+This submodule leverages the [rest\_oai\_pmh module](https://www.drupal.org/project/rest_oai_pmh)
+to provide an OAI-PMH endpoint to allow harvesting repository content.
+
+By default the OAI-PMH endpoint is located at 'yoursite.example/oai/request'
+and includes a set identified as 'oai\_pmh:all\_repository\_items' which can be
+further configured by editing the provided OAI-PMH view. The provided set 
+includes all items using the 'repository item' content model that _don't_ use 
+the 'Collection' model. Additional sets can be created by making views with the
+Entity Reference view display mode and enabling them on the rest\_oai\_pmh
```

Comment:
Should probably say "'repository item' content *type* that don't use the 'Collection' model.".